libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -853,7 +853,10 @@ void Aircraft::update_wind(const struct sitl_input &input)
                        sinf(radians(input.wind.direction))*cosf(radians(input.wind.dir_z)), 
                        sinf(radians(input.wind.dir_z))) * input.wind.speed;
 
-    wind_ef.z += get_local_updraft(position + home.get_distance_NED_double(origin));
+    auto const curr_pos = position + home.get_distance_NED_double(origin);
+    wind_ef.z += get_local_updraft(curr_pos);
+    wind_ef.z += get_topo_lift(location);
```

Comment:
Definitely want this parameterized; don't want more CPU generally - even in SITL :-)

Reply:
Should I parameterize `get_local_updraft` to establish the pattern?